### PR TITLE
created route to fetch user details

### PR DESCRIPTION
**Pull request by olukukoyi**
- pass id into url 

URL: .../api/get-user-details/ USERID

- Fetches row from users table and its relational data

### CODE REVIEW

File: topics-project/app/api/get-user-details/[id]/route.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { NextResponse } from "next/server";
+import { createClient } from "/src/utils/supabase/server";
+
+// pass in userID through params
+// .../api/get-user-details/<userID>
+
+export async function GET(request, context) {
+  const supabase = createClient();
+  const id = context.params.id;
+  console.log(id);
```
Comment (Abdul-Andha) [inline topics-project/app/api/get-user-details/[id]/route.js:10]: Leftover console.log

**Review by Abdul-Andha — APPROVED** [review]
Looks good, just spotted one leftover console.log statement!

**Comment by olukukoyi** [comment]
removed the console log. all good to merge?

**Comment by Abdul-Andha** [comment]
Yup!

**Pull request merged**